pkgs/applications/video/mplayer/default.nix

```
@@ -70,12 +70,11 @@ in
 
 stdenv.mkDerivation rec {
   pname = "mplayer";
-  version = "unstable-2022-02-03";
+  version = "1.5";
 
-  src = fetchsvn {
-    url = "svn://svn.mplayerhq.hu/mplayer/trunk";
-    rev = "38331";
-    sha256 = "1vpic8i6zvg0zsy50vhm45ysqag561bpn9jycfbvvwl9ji7l55zi";
+  src = fetchurl {
+    url = "http://www.mplayerhq.hu/MPlayer/releases/MPlayer-${version}.tar.xz";
```

```suggestion
    url = "https://mplayerhq.hu/MPlayer/releases/MPlayer-${version}.tar.xz";
```

just mimicking url from their homepage